Tools/Scripts/libraries/webkitbugspy/webkitbugspy/bugzilla.py

```diff
@@ -401,6 +401,9 @@ def set(self, issue, assignee=None, opened=None, why=None, project=None, compone
         if keywords is not None:
             update_dict['keywords'] = dict(set=keywords)
 
+        if see_also is not None:
+            update_dict['see_also'] = dict(add=[see_also])
+
```

Comment:
How come you're wrapping this in a list?

I would expect behaviour to be the same as `keywords` above, which also uses `set` and not `add`.

Reply:
There's actually no `set` for `see_also` ([docs](https://bugzilla.readthedocs.io/en/5.0.4/api/core/v1/bug.html#update-bug)). The API takes a list, so either we need to pass in a list or turn a single element into a list. Maybe it would make more sense to pass in a list in the first place  to follow our convention in other functions.

Reply:
> There's actually no set for see_also ([docs](https://bugzilla.readthedocs.io/en/5.0.4/api/core/v1/bug.html#update-bug)).

That's weird and whacky. I don't know what the right approach is here, because I'd rather our API be consistent at least?

> The API takes a list, so either we need to pass in a list or turn a single element into a list. Maybe it would make more sense to pass in a list in the first place to follow our convention in other functions.

I'd suggest we pass in a sequence in the first place, because we might want to add multiple?

Reply:
Yep! I've made that change in the current version